ds3-sdk-samples/src/main/java/com/spectralogic/ds3client/samples/PutEmptyFolderExample.java

```diff
@@ -0,0 +1,90 @@
+ */
+
+package com.spectralogic.ds3client.samples;
+
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.Ds3ClientBuilder;
+import com.spectralogic.ds3client.commands.PutObjectRequest;
+import com.spectralogic.ds3client.commands.PutObjectResponse;
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.networking.FailedRequestUsingMgmtPortException;
+import com.spectralogic.ds3client.utils.EmptySeekableByteChannel;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+import java.security.SignatureException;
+
+/**
+ * Put a zero-byte object with a trailing slash to serve as an empty folder.
+ * Use the Standard Amazon S3 Put Object (does not require a Job)
+ *
+ * NOTE: This PutObjectRequest construstor is marked deprecated
+ * This is to drive development toward Spectra S3 calls; support for this will not be removed
+ */
+public class PutEmptyFolderExample {
+
+    public static void main(final String args[]) throws IOException, SignatureException {
+
+        if (args.length < 2) {
+            System.out.println("Usage: bucketname foldername");
+            return;
+        }
+        final String bucketname = args[0];
+        String foldername = args[1];
+
+        // an object name with trailing / is treated as a folder
+        if (!foldername.endsWith("/")) {
+            foldername += "/";
+        }
+
+        // Get a client builder and then build a client instance.  This is the main entry point to the SDK.
+        try (final Ds3Client client = Ds3ClientBuilder.fromEnv().withHttps(false).build()) {
+
+            // Put using EmptySeekableByteChannel, zero-length
+            final PutObjectResponse putObjectResponse = client.putObject(new PutObjectRequest(bucketname, foldername, new EmptySeekableByteChannel(), 0L));
+
+            // Catch unknown host exceptions.
+        } catch (final UnknownHostException e) {
+
+            System.out.println("Invalid Endpoint Server Name or IP Address");
+
+            // Catch unknown host exceptions.
+        } catch (final FailedRequestUsingMgmtPortException e) {
+
+            System.out.println("Attempted data access on management port -- check endpoint");
+
+            // Catch failed requests with unexpected status codes.
+        } catch (final FailedRequestException e) {
+
+        	// If this is invalid authorization.
+        	if (e.getStatusCode() == 403) {
+                System.out.println("ERROR: Invalid Access ID or Secret Key");
+
+            } else if (e.getStatusCode() == 409) {
+                System.out.println("ERROR: Object Exists");
+
+            // Else unexpected status code.
+            } else {
+        		System.out.println("BlackPearl returned an unexpected status code");
+        	}
+
+        } catch (final IOException e) {
+
+                System.out.println("Encountered a networking error");
+
+        }
```

Comment:
Add some extra code to show that the directory was created and how to interact with it.  A good example would be to list a bucket with the delimiter set to / and printing out the common prefixes.